camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/KafkaReader.java

```diff
@@ -51,7 +52,8 @@
   /**
    * Construct using the json representation of the kafka request
    */
-  public KafkaReader(TaskAttemptContext context, EtlRequest request, int clientTimeout, int fetchBufferSize)
+  public KafkaReader(EtlInputFormat inputFormat, TaskAttemptContext context, EtlRequest request, 
```

Comment:
1. Is 5 parameters too much? Maybe use builder pattern?
2. Instead of passing `EtlInputFormat inputformat`, should we pass `Class<? extends InputFormat> inputFormatClass`, then do `inputFormatClass.newInstance().createSimpleConsumer(...)'?